worker-pools.yml

```diff
@@ -60,6 +60,8 @@ worker-defaults:
             disableReboots: false
             downloadsDir: Z:\downloads
             ed25519SigningKeyLocation: C:\generic-worker\ed25519-private.key
+            enableD2G: false
```

Comment:
I don't think this is doing what you think it is. This is the default config for `generic-worker/windows`. At the very least you'll want this in the other sections here, and perhaps to remove `enableD2G` from this one?